subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -116,10 +116,27 @@ config BT_CTLR_TIFS_HW_SUPPORT
 config BT_CTLR_ULL_LLL_PRIO_SUPPORT
 	bool
 
+config BT_CTLR_RX_POLL
+	# Hidden, Enable Async I/O Framework, i.e. k_poll
+	bool
+	depends on BT_HCI_RAW || BT_HCI_ACL_FLOW_CONTROL
+	select POLL
+	default y
+
 config BT_CTLR_RX_PRIO_STACK_SIZE
-	# Hidden, Controller's Co-Operative high priority Rx thread stack size.
+	# Hidden
```

Comment:
I've seen other reviewers requesting that we don't put e.g. `# hidden` (and even some who have removed them later). 

I'm OK with it as is, but just a note :)

Reply:
This is a comment that makes it obvious of a hidden Kconfig when reading through this file when it appears similar to other non-hidden Kconfig that have `depends on`, `default` and `help`.